desktop/src/main/java/bisq/desktop/main/content/user/user_profile/create/CreateUserProfileController.java

```diff
@@ -75,4 +77,10 @@ protected Optional<? extends Controller> createController(NavigationTarget navig
     public void onQuit() {
          serviceProvider.getShutDownHandler().shutdown();
     }
+
+    void onClose() {
+        Navigation.navigateTo(NavigationTarget.MAIN);
```

Comment:
I guess thats not needed. Of if it should navigate to the user profile target.

Reply:
It works even though it was pointing to Main. But will adjust here